.travis.yml

```diff
@@ -0,0 +1,49 @@
+sudo: false
+
+sudo: required
+dist: trusty
+
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+      - graphviz
```

Comment:
It's a dependency of the vulkan SDK.  Once there's a vulkan SDK package in the distro this will get a lot simpler, and no longer need sudo.  There's not even a way to install the vulkan SDK locally without root right now (using the lunarg installer anyway).
> @@ -0,0 +1,49 @@
> +sudo: false
> +
> +sudo: required
> +dist: trusty
> +
> +addons:
> +  apt:
> +    packages:
> +      - libgmp-dev
> +      - graphviz

Why does this depend on graphviz?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/expipiplus1/vulkan/pull/11/files/19c6abfc3ae3b4f78484b4803e5e6db1d2686e09#r58128790